From 14a545f9a5c2748d293cd379173617762ab10403 Mon Sep 17 00:00:00 2001
From: Aleksander Morgado <aleksander@aleksander.es>
Date: Thu, 27 Oct 2016 18:48:52 +0200
Subject: [PATCH 1/7] libqmi-glib,proxy: print error message when indication
 cannot be forwarded

(cherry picked from commit 13747cf3b03cb288e87c83779dc8c2320ea0ea0e)
---
 src/libqmi-glib/qmi-proxy.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/src/libqmi-glib/qmi-proxy.c b/src/libqmi-glib/qmi-proxy.c
index 0731f37..c5e936e 100644
--- a/src/libqmi-glib/qmi-proxy.c
+++ b/src/libqmi-glib/qmi-proxy.c
@@ -280,7 +280,7 @@ indication_cb (QmiDevice *device,
             GError *error = NULL;
 
             if (!send_message (client, message, &error)) {
-                g_warning ("couldn't forward indication to client");
+                g_warning ("couldn't forward indication to client: %s", error->message);
                 g_error_free (error);
             }
 
-- 
2.10.1

